ingestion_server/ingestion_server/api.py

```diff
@@ -87,17 +87,19 @@ def on_post(self, req, resp):
         # Inject shared memory
         progress = Value("d", 0.0)
         finish_time = Value("d", 0.0)
+        active_workers = Value("i", int(False)) # Tracks whether the task has any active distributed workers
```

Comment:
Following the model here from `progress` and `finish_time`, although this is new to me.